--- /usr/lib/obs/server/BSConfig.pm.orig       2020-04-29 19:34:39.000000000 +0000
+++ /usr/lib/obs/server/BSConfig.pm    2020-04-30 10:05:14.000000000 +0000
@@ -32,7 +32,7 @@
 my $frontend = undef; # FQDN of the WebUI/API server if it's not $hostname
 
 # If defined, restrict access to the backend servers (bs_repserver, bs_srcserver, bs_service)
-our $ipaccess = {
+our $removed_ipaccess  = {
    '^::1$' => 'rw',    # only the localhost can write to the backend
    '^127\..*' => 'rw', # only the localhost can write to the backend
    "^$ip\$" => 'rw',   # Permit IP of FQDN
@@ -49,9 +49,9 @@
 #our $ssl_certfile = "/path/to/tls.pem";
 
 # Change also the SLP reg files in /etc/slp.reg.d/ when you touch hostname or port
-our $srcserver = "http://$hostname:5352";
-our $reposerver = "http://$hostname:5252";
-our $serviceserver = "http://$hostname:5152";
+our $srcserver = "http://srcserver:5352";
+our $reposerver = "http://repserver:5252";
+our $serviceserver = "http://serviceserver:5152";
 our $clouduploadserver = "http://$hostname:5452"; 
 
 # you can use different ports for worker connections
@@ -250,7 +250,7 @@
 #our $modifyrepo = '/usr/bin/modifyrepo_c';
 
 # enable service dispatcher
-our $servicedispatch = 1;
+our $servicedispatch = 0;
 # max of 4 parallel running services (default)
 # our $servicedispatch_maxchild = 4;
